reference/features/search_parameters.md

```diff
@@ -150,6 +150,20 @@ You can then use the filter in a search query:
 
 <CodeSamples id="faceted_search_walkthrough_filter_1" />
 
+### Filtering results `_geoRadius`
+
+If your documents contain `_geo` data, you can use the `_geoRadius` built-in filter rule to filter results according to their their geographic position.
```

Comment:
If your documents contain `_geo` data, you can use the `_geoRadius` built-in filter rule to filter results according to their geographic position.